specifyweb/frontend/js_src/lib/components/ChooseCollection/index.tsx

```diff
@@ -125,7 +125,7 @@ function Wrapped({
         {errors.length > 0 && <ErrorMessage>{errors}</ErrorMessage>}
         {hasAccess ? (
           <>
-            <div className="-ml-1 flex max-h-56 flex-col gap-2 overflow-y-auto pl-1">
+            <div className="-ml-1 flex max-h-[50vh] flex-col gap-2 overflow-y-auto pl-1">
```

Comment:
This would be bad on mobile in the landscape. Can you disable it on less tall screens? Or leave it in rem like it was, but increase the number

Reply:
Edit:
It does look fine actually. I didn't realize how much space was taken by logo, label and button, so collection switcher being just 50% of the screen is fine